portfolio/src/main/webapp/style.css

```diff
@@ -158,12 +154,54 @@ padding-left: 25px
   width: 40%;
 }
 
-.effect_text {
+.effect_text{
```

Comment:
nit: These spaces should be left in per the style guide: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation

Reply:
Got it